build/chip/java/rules.gni

```diff
@@ -16,11 +16,25 @@ import("//build_overrides/chip.gni")
 
 import("${chip_root}/build/config/android/config.gni")
 
+java_path = getenv("JAVA_PATH")
```

Comment:
java_path could be detected automatically from the current environment, but we can do it in the follow-up